godot-codegen/src/util.rs

```diff
@@ -65,7 +93,7 @@ pub fn get_api_level(class: &JsonClass) -> ClassCodegenLevel {
             )
     }
 
-    if class.name.ends_with("Server") {
+    if class_api_level_is_server(&class.name) {
```

Comment:
Then this could be `special_cases::is_class_level_server` (explicitly specifying module is important).